test-it/karma/javascriptinvoke/Async.Test.js

```diff
@@ -0,0 +1,188 @@
+describe('A JavaScript function invoke', function () {
+    'use strict';
+    // Reference aliases
+    var Vireo = window.NationalInstruments.Vireo.Vireo;
+    var vireoRunner = window.testHelpers.vireoRunner;
+    var fixtures = window.testHelpers.fixtures;
+
+    var vireo;
+
+    var jsAsyncFunctionsUrl = fixtures.convertToAbsoluteFromFixturesDir('javascriptinvoke/AsyncFunctions.via');
+    var javaScriptInvokeFixtures = {
+        NI_AsyncSquareFunction: function (inputInteger) {
+            var completionCallback = this.getCompletionCallback();
+            setTimeout(function () {
+                completionCallback(inputInteger * inputInteger);
+            }, 4);
```

Comment:
Like above it is better to declare your intent. You want this to run an asynchronous task as quickly as possible, ie setTimeout(fn, 0);

Today the browsers internally use a minimum of 4ms, but maybe tomorrow it will drop and our tests will run quicker.

Reply:
ommitting it is fine as well like below, I would say just be consistent https://github.com/ni/VireoSDK/search?l=JavaScript&q=setTimeout&type=